planex/cmd/fetch.py

```diff
@@ -161,7 +161,7 @@ def fetch_url(url, source, retries):
     except requests.RequestException as exn:
         # Download failed
         sys.exit("%s: Failed to fetch %s: %s" %
-                 (sys.argv[0], urlunparse(url), exn.args[1]))
+                 (sys.argv[0], urlunparse(url), exn.message))
```

Comment:
I thought `BaseException.message` had been deprecated

https://www.python.org/dev/peps/pep-0352/#retracted-ideas

Reply:
exn.args[0] would also work. That seems to have had its deprecation retracted (my head is spinning)

Reply:
What's wrong with `str(exn)`?